lgpl-2.1.txt

```diff
@@ -55,7 +55,7 @@ modified by someone else and passed on, the recipients should know
 that what they have is not the original version, so that the original
 author's reputation will not be affected by problems that might be
 introduced by others.
-
+
```

Comment:
I could be convinced on this one, but the same argument applies here.  This file is vendored in from elsewhere.